src/data/pages/TEST-size-distance-plotter.json

```diff
@@ -24,6 +24,7 @@
       },
```

Comment:
delete this test page and reimplement this plot on `http://localhost:8000/object-magnitude-solar-system/`.  Also, while you're updating that page, please fix this typo in the last question on that page: "...large object throught the solar system?" should be "...large objects throughout the solar system?"